translation-service-google/src/main/java/eu/europeana/api/translation/service/google/GLangDetectThreshold.java

```diff
@@ -0,0 +1,46 @@
+package eu.europeana.api.translation.service.google;
+
+/**
+ * Definition of a confidence threshold applicable to a text length range
+ * 
+ * @author Nuno Freire
+ * @since 29/01/2025
+ */
+public class GLangDetectThreshold {
```

Comment:
I think this class is not used anywhere and it is a duplicate of the ThresholdByTextLength. If so, please delete.